dali/kernels/transpose/transpose_gpu_impl.cuh

```diff
@@ -100,9 +100,9 @@ namespace transpose_shared {
   extern __shared__ uint8_t shared_tmp[];
 }  // namespace transpose_shared
 
-
-template <int ndim, typename T>
-__device__ void TransposeTiledStatic(TiledTransposeDesc<T> desc) {
+template <int ndim, unsigned pack_ratio, typename T, typename OldT>
```

Comment:
Nitpick: pack_ratio is not really a ratio, but rather a pack_ratio_log - renaming or a commend would be nice (but it's not a must).

Reply:
changed